### PR TITLE
Made TimerTest a Fixture instance and fixed issues on Windows

### LINKED ISSUE #715: Timer and Profiler do not work on Windows
As I was writing a test for `Timer` and `ClockTimer` (#714) I noticed that it kept failing on AppVeyor. I have now confirmed that `Timer` does indeed not work on Windows at all - which means `Profiler` also doesn't work.

The reason is that `Time runTimeMicro()` always returns `UINT_MAX` on Windows.

`ClockTimer` works well because it uses a different implementation.

### PR DESCRIPTION
**Pull request by marcusnaslund**
Not a great test, but certainly better than the original one which didn't really do anything.

Fixes #715

### CODE REVIEW

**Comment by marcusnaslund** [comment]
Interesting. It would appear that `Timer` does not actually work on Windows.

File: source/sdk/os/Time.ooc [hunk 1]
```diff
@@ -145,6 +145,9 @@ Time: class {
             gettimeofday(tv&, null)
             return ((tv tv_usec + tv tv_sec * 1_000_000) - __time_millisec_base) as UInt
         }
+		version(windows) {
+            return This runTime() * (1000 as UInt)
+        }
```
Comment (davidhesselbom) [inline source/sdk/os/Time.ooc]: The indentation is messed up here, but since it's in the SDK, I'm not too bothered.

Comment (marcusnaslund) [inline source/sdk/os/Time.ooc]: It's at least the same as three lines above. :)

Comment (davidhesselbom) [inline source/sdk/os/Time.ooc]: No, line 148 uses tabs, line 150 uses spaces.

Comment (marcusnaslund) [inline source/sdk/os/Time.ooc]: Yes, but compare with line 143 and 147.

Comment (davidhesselbom) [inline source/sdk/os/Time.ooc]: Oh, right. Well, both are wrong, but I'm still not bothered.

**Comment by davidhesselbom** [comment]
@sebastianbaginski Any objections?

**Comment by sebastianbaginski** [comment]
@davidhesselbom no, it's better than before

**Pull request merged**